src/Modal/Portal.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import PropTypes from "prop-types";
+import cx from 'classnames';
+import Panel from '../Panel/Panel';
+import styles from "./Modal.module.scss";
+
+class Portal extends React.Component {
+  constructor() {
+    super();
+
+    this.mountNode = document.createElement('div');
+    this.mountNode.id = 'ui-modal';
+    this.mountNode.className = styles.wrapper;
+    document.body.appendChild(this.mountNode);
+  }
+
+  componentDidMount() {
+    const dimmer = this.mountNode.getElementsByClassName('dimmer')[0];
+    dimmer.focus();
+  }
+
+  handleOnKeyUp = e => {
+    if (e.keyCode == 27) {
+      this.props.closeModal();
+    }
+  };
+
+  componentWillUnmount() {
+    document.body.removeChild(this.mountNode);
+  }
+
+  render() {
+    const { children, closeModal } = this.props;
+
+    return ReactDOM.createPortal(
+      <React.Fragment>
+        <Panel variant="modal" className={styles.dialog}>
+          {children}
+        </Panel>
+        <div
+          tabIndex={0}
```

Comment:
Unnecessary prop

Reply:
Without `tabIndex` the onKeyUp event doesn't receives the `keyCode`.